assets/src/edit-story/components/panels/design/borderRadius/borderRadius.js

```diff
@@ -143,51 +175,55 @@ function BorderRadiusPanel({ selectedElements, pushUpdateForObject }) {
     : __('Top left corner radius', 'web-stories');
   return (
     <SimplePanel name="borderRadius" title={__('Corner radius', 'web-stories')}>
-      <BorderRow>
-        <BoxedNumeric
-          value={borderRadius.topLeft}
-          aria-label={firstInputLabel}
-          onChange={(value) => handleChange('topLeft', value)}
-        />
-        {!lockRadius && (
-          <>
-            <Icon>
-              <Union />
-            </Icon>
-            <BoxedNumeric
-              value={borderRadius.topRight}
-              aria-label={__('Top right corner radius', 'web-stories')}
-              onChange={(value) => handleChange('topRight', value)}
+      <FlexContainer>
+        <InputContainer>
+          <Row>
+            <BoxedNumericInput
+              value={borderRadius.topLeft}
+              aria-label={firstInputLabel}
+              onChange={(_, value) => handleChange('topLeft', value)}
+            />
+            {!lockRadius && (
+              <>
+                <Icon>
+                  <Union />
+                </Icon>
+                <BoxedNumericInput
+                  value={borderRadius.topRight}
+                  aria-label={__('Top right corner radius', 'web-stories')}
+                  onChange={(_, value) => handleChange('topRight', value)}
+                />
+              </>
+            )}
+          </Row>
+          {!lockRadius && (
+            <Row>
+              <BoxedNumericInput
+                value={borderRadius.bottomLeft}
+                aria-label={__('Bottom left corner radius', 'web-stories')}
+                onChange={(_, value) => handleChange('bottomLeft', value)}
+              />
+              <Space space={32} />
+              <BoxedNumericInput
+                value={borderRadius.bottomRight}
+                aria-label={__('Bottom right corner radius', 'web-stories')}
+                onChange={(_, value) => handleChange('bottomRight', value)}
+              />
+            </Row>
+          )}
+        </InputContainer>
+
+        <LockContainer>
```

Comment:
There is a separate Lock toggle button variant in the design system, [see the storybook](https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-components-button--prebaked-buttons).

Reply:
Oh awesome!